ci-scripts/update-all-on-remote.sh

```diff
@@ -1,3 +1,5 @@
+set -e
```

Comment:
Is this still needed?

Reply:
I'm fairly certain it is, from what I was seeing. Its a hotfix for early exit when something fails.